modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/internal/layout/admin/util/CPDefinitionSitemapURLProvider.java

```diff
@@ -161,13 +169,35 @@ protected void visitLayout(
 				_portal.getClassNameId(CProduct.class),
 				cpDefinition.getCProductId());
 
-		String productFriendlyURL =
-			currentSiteURL + urlSeparator +
-				friendlyURLEntry.getUrlTitle(themeDisplay.getLanguageId());
+		String productFriendlyURL = StringBundler.concat(
+			currentSiteURL, urlSeparator,
+			friendlyURLEntry.getUrlTitle(themeDisplay.getLanguageId()));
+
+		List<FriendlyURLEntryLocalization> friendlyURLEntryLocalizations =
+			_friendlyURLEntryLocalService.getFriendlyURLEntryLocalizations(
+				friendlyURLEntry.getFriendlyURLEntryId());
+
+		Map<Locale, String> alternateFriendlyURLs = new HashMap<>();
+
```

Comment:
```suggestion
		Map<Locale, String> alternateFriendlyURLs = new HashMap<>();
		
		List<FriendlyURLEntryLocalization> friendlyURLEntryLocalizations =
			_friendlyURLEntryLocalService.getFriendlyURLEntryLocalizations(
				friendlyURLEntry.getFriendlyURLEntryId());

```

I meant in this order, https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DLocal-variable-declarations

Reply:
@andrea-ale-sbarra , thank you for letting me know about that page. I will take a deep look at it. I updated the PR accordingly.